module/Finna/src/Finna/View/Helper/Root/Piwik.php

```diff
@@ -81,7 +88,14 @@ public function __invoke($params = null)
             }
         }
 
-        return parent::__invoke($params);
+        // Passed custom variables are stored for this call only.
+        if (isset($params['customVars'])) {
+            $this->passedCustomVars = $params['customVars'];
```

Comment:
Tätä ei tarvita. parent::__invoke asettaa $params => $this->params, jolloin kutsun customVars:t voi lukea sieltä. Muuten pitäisi catchata poikkeukset ym. jotta passedCustomVars nollattaisiin oikeaoppisesti.

Reply:
No niinpä tekee, loistavaa.